README.md

```diff
@@ -204,17 +329,26 @@ It is our modified version of the official brightscript testing framework provid
 Just make sure that your test component always extends the component you want to test.
 
 ### Initializing the component/initial props
```

Comment:
I would also add somewhere the initialization order of the children's components. Parent -> children, so to clearly state which lifecycle methods are invoked first.